### PR TITLE
duktape: use github releases & install duk binary

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/duktape.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Duktape < Formula
   desc "Embeddable Javascript engine with compact footprint"
   homepage "https://duktape.org"
-  url "https://duktape.org/duktape-2.5.0.tar.xz"
```
Comment (chenrui333) [inline Formula/duktape.rb:4]: why do switch to use github source?

Comment (vladimyr) [reply Formula/duktape.rb:4]: We absolutely don't but it enables brew livecheck to check for upstream updates.

Comment (chenrui333) [reply Formula/duktape.rb:4]: i see, that makes sense.

**Review by chenrui333 — COMMENTED** [review]
LGTM 👍

**Review by SMillerDev — DISMISSED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @vladimyr!

**Pull request closed without merging**